packages/block-library/src/comment-author-name/index.php

```diff
@@ -28,7 +28,7 @@ function render_block_core_comment_author_name( $attributes, $content, $block )
 		$classes .= 'has-text-align-' . esc_attr( $attributes['textAlign'] );
 	}
 
-	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classes ) );
+	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => esc_attr( $classes ) ) );
```

Comment:
`get_block_wrapper_attributes` already does the escaping.

Reply:
Yup agreed. I've been going through the other PRs removing this as well.

Reply:
✅